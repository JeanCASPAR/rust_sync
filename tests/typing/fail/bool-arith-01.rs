use rust_sync::sync;

sync! {
    #![pass(1)]
    
    node oui() = (a)
    where
        a : int = -false;
}

fn main() {}
