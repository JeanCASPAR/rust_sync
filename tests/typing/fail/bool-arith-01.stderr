error: type mismatch: expected `int`, found `bool`
 --> tests/typing/fail/bool-arith-01.rs:6:19
  |
6 |         a : int = -false;
  |                   ^^^^^^
